cylc/flow/host_select.py

```diff
@@ -543,7 +543,7 @@ def _get_metrics(hosts, metrics, data=None):
             if proc.poll() is None:
                 continue
             del proc_map[host]
-            out, err = (f.decode() for f in proc.communicate())
+            out, err = proc.communicate()
```

Comment:
From a look at `run_cmd` these processes don't seem to be opened in text mode?

Reply:
I had a look
https://github.com/cylc/cylc-flow/blob/d08a659c83aaeab9ea3b41a462aece857b7e492d/cylc/flow/host_select.py#L536-L538

Both `_remote_cylc_cmd()` and `run_cmd()` have `text=True` default arg.